pkgs/by-name/ro/rose-pine-cursor/package.nix

```diff
@@ -0,0 +1,39 @@
+{ lib
+, stdenvNoCC
+, fetchurl
+}:
+
+stdenvNoCC.mkDerivation (finalAttrs: {
+  pname = "rose-pine-cursor";
+  version = "1.1.0";
+
+  srcs = [
+    (fetchurl {
+      url = "https://github.com/rose-pine/cursor/releases/download/v${finalAttrs.version}/BreezeX-RosePine-Linux.tar.xz";
+      hash = "sha256-szDVnOjg5GAgn2OKl853K3jZ5rVsz2PIpQ6dlBKJoa8=";
+    })
+    (fetchurl {
+      url = "https://github.com/rose-pine/cursor/releases/download/v${finalAttrs.version}/BreezeX-RosePineDawn-Linux.tar.xz";
+      hash = "sha256-hanfwx9ooT1TbmcgCr63KVYwC1OIzTwjmxzi4Zjcrdg=";
+    })
+  ];
+
+  sourceRoot = ".";
+
+  dontConfigure = true;
+  dontBuild = true;
```

Comment:
You can get rid of these, the default configure and build phases will not do anything if there is no configure script/makefile.

Reply:
I am aware, but it's unecessary if it's not used, but if you want me to remove it just confirm it

Reply:
Yeah please remove it, these "automatic phases" exist so the packages that need to use them don't have to specify those extra phases, while the packages that do not need to use them can just leave them alone without trouble. Smaller is better :)